tests/reg_tests/test_DVGeometryESP.py

```diff
@@ -0,0 +1,654 @@
+import unittest
+from mpi4py import MPI
```

Comment:
This is going to raise an error if `mpi4py` is not available. Consider wrapping with try/except as you did with pyOCSM.

Reply:
Fixed!